### ISSUE TITLE
color contrast

### ISSUE THREAD

**Issue by brianteeman** [akeeba/panopticon#78]
Maybe you didnt spot it because you use dark mode but the highlighted colours in this screenshot all fail a11y contrast checks. Especially the count of extensions

![image](https://github.com/akeeba/panopticon/assets/1296369/9d67275e-a1e4-48ff-acd1-1eb9e0e3c0d8)

**Comment by nikosdion** [comment]
I am actually toggling between light and dark mode on a time schedule :)

The main orange/yellow and white do, indeed, appear to fail the WCAG color contrast test… but you should note that it is based on _the wrong color model_ which fails on very light and very dark colors. This has already been discussed and addressed for WCAG 3 which uses a different, correct color model. Under this model, the contrast passes.

The little badge next to Extensions does fail under both models, I am aware of this. It's not even really legible for me. It's on my long tail of minor issues to fix. I say it's minor because the information is duplicated in the main table area, so if you can't see it it's not the end of the world.

**Comment by brianteeman** [comment]
yes I'm aware of all those contrast changes. I was only really bothered about the extensions count because I coouldnt see it at all ;)

as your long tail is invisible to me I can only report what I can see and let you do the rest ;)
